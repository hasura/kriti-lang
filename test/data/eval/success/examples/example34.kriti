{{ if $[0].isActive }}
   "if_evaluated"
{{ elif (not $[1].isActive) }}
   "elif_evaluated"
{{ elif (not $[0].isActive) }}
   "another_elif_evaluated"
{{ else }}
   "none_evaluated"
{{ end }}