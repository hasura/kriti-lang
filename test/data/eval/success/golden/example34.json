"another_elif_evaluated"